framework/tst/dslabs/framework/testing/newviz/SingleNodePanel.java

```diff
@@ -137,6 +145,11 @@ void updateState(EventTreeState s, boolean viewDeliveredMessages) {
             nodeState.clearDiffObject();
         }
 
+        boolean searchDone = settings != null &&
+                                (settings.invariantViolated(s.state()) ||
+                                 settings.shouldPrune(s.state()) ||
```

Comment:
See previous comment.

Reply:
Fixed in https://github.com/emichael/dslabs/pull/29/commits/c0f1eba7cc6966e25eeefa893dd43ac666d0c7a8.